components/api/core/utils.bal

```
@@ -39,3 +40,81 @@ function updatePayloadWithUserInfo (json payload, string field) returns error? {
         log:printDebug(io:sprintf("Response modification failed : User information not found for user : \'%s\'", userId));
     }
 }
+
+function getUserToken(string authorizationHeader, string cookieHeader) returns string {
+    string token = "";
+    string[] splittedToken = authorizationHeader.split(" ");
+    if splittedToken.length() != 2 || !(splittedToken[0].equalsIgnoreCase(constants:BEARER_HEADER)) {
+        log:printError("Did not receive the token in proper format");
```

Shouldn't we return an error here and break the flow ?